reinforcement_learning/rlclientlib/schema/OutcomeEvent.fbs

```diff
@@ -0,0 +1,12 @@
+﻿// EventHubInteraction Schema used by FlatBuffer
+
+namespace VW.Events;
+
+table OutcomeEvent {
```

Comment:
Can we use c++ style naming convention please?  This library is external facing and would be good to not have to switch to different styles for generated code.